### ISSUE TITLE
Bypass the Unreal Engine networking layer using Unreal.js?

### ISSUE THREAD

**Issue by HeadClot** [ncsoft/Unreal.js#156]
Hey,
So - I am looking at bypassing the Unreal networking layer using Unreal.JS and replacing it using [Incheon](https://github.com/OpherV/Incheon).

Here are my questions -
1. Would bypassing the default network layer be possible to do currently with Unreal.js?
2. Would bypassing the default UE Networking layer be desirable? 
3. Does Unreal.js play nice with Node.js?

**Comment by Youdaman** [comment]
I'm not the developer, but I would say:

1. you could use Incheon with Unreal.js if you wanted
2. you'd probably be better off just using UE Networking
3. yes you can use Node modules because Unreal.js is also built on v8

**Comment by HeadClot** [comment]
Hey @Youdaman Thank you for the reply :)

You are probably right with using the UE networking opposed to just bypassing it. There is allot of functionality built into it.

**Comment by Youdaman** [comment]
@HeadClot yeah using the built-in networking means things that should get updated will get updated, especially if you're using its replication stuff.
